api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -836,7 +837,7 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
         let row = self
             .tx
             .query_opt(
-                r#"SELECT pool_id, balance, spend_destination, next_nonce
+                r#"SELECT pool_id, balance, spend_destination, next_nonce, creation_time
```

Comment:
Perhaps creation block height makes more sense... because the time is derived from that?